agent/roles/manifests/templates/install-config_yaml.j2

```diff
@@ -44,7 +44,7 @@ networking:
 {% endif %}
   networkType: {{ network_type }} 
 platform:
-{% if num_masters == "1" %}
+{% if (num_masters == "1") or (platform_type == "none") %}
```

Comment:
When using SNO could automatically set PLATFORM_TYPE=none in common.sh which would simplify this check.